lib/Basics/StringUtils.cpp

```diff
@@ -1963,6 +1963,37 @@ std::string escapeRegexParams(std::string const& in) {
   return out;
 }
 
+std::string formatSize(uint64_t value) {
+  std::string out, label;
+  if (value < 1000) {
+    if (value == 1) {
+      out = "1";
+      label ="byte";
+    } else {
+      out = std::to_string(value);
+      label = "bytes";
+    }
+  } else if (value < 1000000ULL) {
+    out = std::to_string((double) value / double(1000));
```

Comment:
We seem to use a mix of binary and decimal prefix units:
https://www.arangodb.com/docs/stable/administration-configuration.html#suffixes-for-numeric-options
Would it make sense to use a factor of 1024 and KiB, MiB, etc. here?

Reply:
Yes, this would also work.
But I am not clear about why you sent me this docs link, because it lists _both_ types of units (base 10 and base 2) as valid and supported. So I am not clear why we should go with base 2 outputs here and not base 10.
Regardless of for which units (base 2 or base 10) we decide here, there will be someone who doesn't like it.

Reply:
The base 2 variant is what I perceive as "correct" for everything computer-related. Base 10 doesn't make much sense to me, but unfortunately drive capacity is specified using base 10. I'm not sure how different OSes handle it, but under Windows, a 1 TB drive shows up with a capacity of ~930 GB (actually GiB) for instance. Personally, I'm in favor of base 2, but as you say, people have different opinions and we already have both bases in use in ArangoDB, so it's unclear what the preferred one should be. The binary prefixes are at least less ambiguous (see Windows).